### PR TITLE
Feature/86cv9ra0w availability form

### PR DESCRIPTION
**Pull request by laurazhakupova**

### CODE REVIEW

File: behind-the-veil-siteroot/imports/ui/components/availabilityCalendar/AvailabilityCalendar.css
Comment (joshualoongwy) [inline behind-the-veil-siteroot/imports/ui/components/availabilityCalendar/AvailabilityCalendar.css:1]: I like the name choice

File: behind-the-veil-siteroot/package-lock.json
Comment (joshualoongwy) [inline behind-the-veil-siteroot/package-lock.json:1]: What version of npm are you using when running 'npm install' (can check by running npm -v)? I think ideally we want to keep the lockfileVersion the same as it currently is in the develop branch (i.e. lockfileVersion 1)

File: behind-the-veil-siteroot/imports/ui/components/pages/add-availability/AddAvailability.jsx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+                isValid(parsedDate) &&
+                isWithinInterval(parsedDate, VALID_INTERVAL)
+            ) {
+                setInputs((i) => ({...i, date: parsedDate}));
+                return
+            }
+        }
+
+        // else, just update date input with the raw value
+        //setInputs((i) => ({...i, date: dateInput}));
```
Comment (joshualoongwy) [inline behind-the-veil-siteroot/imports/ui/components/pages/add-availability/AddAvailability.jsx:127]: Is this supposed to be commented out? Without this line, the manual date input logic doesn't work

Comment (laurazhakupova) [reply behind-the-veil-siteroot/imports/ui/components/pages/add-availability/AddAvailability.jsx]: It was not supposed to, I have updated it

**Review by joshualoongwy — DISMISSED** [review]

**Review by joshualoongwy — APPROVED** [review]

**Pull request merged**